### PR TITLE
Add a webhook service for CircleCI builds

### PR DESCRIPTION
**Pull request by EdJoJob**
Notify chanels when builds fail or recover for important branches.

### CODE REVIEW

File: scripts/circle-ci-notifications.coffee [hunk 1]
```diff
@@ -0,0 +1,26 @@
+watchers = {
+  "besu": [
+      {regex: /master/, channels: ["besu-contributors"]},
+      {regex: /release-.*/, channels: ["besu-contributors"]}
```
Comment (joshuafernandes) [inline scripts/circle-ci-notifications.coffee]: Would be worth creating a default_channel="besu-contributors" and using that var

Comment (EdJoJob) [reply scripts/circle-ci-notifications.coffee]: I have added f923fc2caeec2904de652c5622c352214f20b78f as an alternative to multiple channels for a build branch regex.  Does that fit better with your idea?

Comment (joshuafernandes) [reply scripts/circle-ci-notifications.coffee]: :+1:

**Comment by ryjones** [comment]
Could I ask you to squash this into one commit on the head of master? thanks

**Comment by EdJoJob** [comment]
> Could I ask you to squash this into one commit on the head of master? thanks

Will do when it is done.  I had a re-realization that I need to include the Job name in the brain cache as CircleCI sends the status of each individual job in the workflow, and never the status for the workflow as a whole

**Comment by ryjones** [comment]
let me know when you're ready for this to merge

**Comment by EdJoJob** [comment]
@ryjones this PR is now ready to merge

**Pull request merged**